kubernetes/cmsweb/services/reqmon.yaml

```diff
@@ -58,8 +58,8 @@ spec:
         name: reqmon
         resources:
           requests:
-            memory: "256Mi"
-            cpu: "300m"
+            memory: "5Gi"
```

Comment:
The same comments as with reqmon-stats. This limit defines how much RAM required to start the app. I doubt we need 5Gi for that.

Reply:
same comment as above (the service itself runs with the DataCache thread enabled, thus high memory consumption).

Reply:
ok, could you please add an appropriate comment over there, otherwise we'll forget about it.

Reply:
comment added.